notebooks/eight_schools.ipynb

```diff
@@ -0,0 +1,533 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stan NUTS inference \n",
+    "\n",
+    "This allows us to run the Stan code (make sure pystan is installed for this step)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:pystan:COMPILING THE C++ CODE FOR MODEL anon_model_9d66c37004ddc3041422a88e70f9afbc NOW.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Inference for Stan model: anon_model_9d66c37004ddc3041422a88e70f9afbc.\n",
+      "4 chains, each with iter=100000; warmup=50000; thin=1; \n",
+      "post-warmup draws per chain=50000, total post-warmup draws=200000.\n",
+      "\n",
+      "                 mean se_mean     sd   2.5%    25%    50%    75%  97.5%  n_eff   Rhat\n",
+      "mu                5.8    0.02   5.44  -5.35   2.34   5.93   9.41  16.23  95402    1.0\n",
+      "logtau           2.45  1.9e-3   0.51   1.38   2.12   2.47   2.79    3.4  73132    1.0\n",
+      "theta_tilde[0]   0.65  1.8e-3    0.8  -0.96   0.14   0.66   1.18   2.22 200000    1.0\n",
+      "theta_tilde[1]   0.09  1.6e-3    0.7  -1.32  -0.35    0.1   0.54   1.49 200000    1.0\n",
+      "theta_tilde[2]  -0.24  1.8e-3   0.81  -1.84  -0.77  -0.23   0.29   1.38 200000    1.0\n",
+      "theta_tilde[3]   0.05  1.6e-3   0.73  -1.42  -0.42   0.05   0.52   1.49 200000    1.0\n",
+      "theta_tilde[4]  -0.34  1.5e-3   0.68  -1.74  -0.76  -0.32    0.1    1.0 200000    1.0\n",
+      "theta_tilde[5]   -0.2  1.6e-3   0.73  -1.66  -0.66  -0.19   0.28   1.24 200000    1.0\n",
+      "theta_tilde[6]   0.54  1.6e-3    0.7  -0.87    0.1   0.54   0.99   1.96 200000    1.0\n",
+      "theta_tilde[7]   0.15  1.9e-3   0.83  -1.52   -0.4   0.16    0.7   1.79 200000    1.0\n",
+      "lp__            -9.35    0.01   2.81 -15.59 -11.06  -9.09  -7.34  -4.64  55356    1.0\n",
+      "\n",
+      "Samples were drawn using NUTS at Thu Feb 15 19:27:02 2018.\n",
+      "For each parameter, n_eff is a crude measure of effective sample size,\n",
+      "and Rhat is the potential scale reduction factor on split chains (at \n",
+      "convergence, Rhat=1).\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pystan\n",
+    "standata = dict(J=J, y=data_y, sigma=data_sigma)\n",
+    "\n",
+    "fit = pystan.stan(model_code=eight_schools_stan, \n",
+    "                  data=standata, iter=100000)\n",
+    "print(fit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The inferred parameter theta_tilda or $\\theta'$ is a transform, we might prefer to convert this back to the more intepretable theta by posthoc processing of the samples or by including a generated quantities block in the Stan code.  In this case the required postprocessing is quite simple."
```

Comment:
Does $\\tilde{\\theta}$ work? Currently you call it theta_tilde but use the symbol for "theta prime".

Reply:
It isn't great, but it is better.. thanks!